app/lib/search/mem_index.dart

```diff
@@ -184,7 +184,8 @@ class InMemoryPackageIndex {
       packageHits = packageHits.map((ps) {
         final apiPages = textResults.topApiPages[ps.package]
             // TODO: extract title for the page
```

Comment:
Nit: Do we want to elaborate(or make an issue?) on this todo since we are touching the surrounding code?